### PR TITLE
Report form error states

### LINKED ISSUE #3547: Reporting Form: error states
add error handling for:

 * input value not between scalar market range
 * invalid REP value (needs to be valid amount)
 * REP more than user balance

Asked design if there ar mocks

### PR DESCRIPTION
**Pull request by phoebemirman**
closes https://github.com/AugurProject/augur/issues/3547

### CODE REVIEW

**Review by bthaile — APPROVED** [review]
lgtm, reporting errors looked good

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 1]
```diff
@@ -149,7 +150,8 @@ export class PreFilledStake extends Component<
 
     const {
       preFilledStake,
-      stakeError
+      stakeError,
+      threshold
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.tsx:150]: Prob should be string. Number could get large

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 2]
```diff
@@ -257,6 +259,7 @@ export interface DisputingBondsViewProps {
   maxPrice?: string;
   rep: number;
   stakeRemaining?: number;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.tsx:257]: All these “big number” values from getters should prob stay strings.

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 3]
```diff
@@ -463,7 +471,8 @@ export class ReportingBondsView extends Component<
       reportingGasFee,
       reportAction,
       preFilledStake,
-      updatePreFilledStake
+      updatePreFilledStake,
+      rep
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.tsx]: If this is userRep, just a suggestion change to ‘userAvailableRep or if its already formatted change to userRepFormatted.

File: packages/augur-ui/src/modules/common/form.tsx [hunk 4]
```diff
@@ -748,6 +752,9 @@ export const ReportingRadioBarGroup = ({
           scalarOutcome={scalarOutcome}
           updateScalarOutcome={updateScalarOutcome}
           onChange={selected => {
+            updateDisputeStake("");
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/form.tsx]: I assume these are default values

File: packages/augur-ui/src/modules/common/form.tsx [hunk 5]
```diff
@@ -790,6 +799,8 @@ export const ReportingRadioBarGroup = ({
             checked={invalid.value.toString() === selected}
             reportAction={reportAction}
             onChange={selected => {
+              updateDisputeStake("");
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/form.tsx]: Seems to be a bunch of them

File: packages/augur-ui/src/modules/common/form.tsx (hunk 4 above)
Comment (phoebemirman) [reply packages/augur-ui/src/modules/common/form.tsx]: yep

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🍉

 This link will always point to this PR's latest changes: https://peek.run/5m3hg27

**Pull request merged**